zenoh-flow/src/runtime/dataflow/instance/runners/connector.rs

```diff
@@ -30,7 +30,7 @@ use zenoh_util::core::AsyncResolve;
 /// different runtimes.
 pub(crate) struct ZenohSender {
     pub(crate) id: NodeId,
-    pub(crate) input: Input,
+    pub(crate) receivers: Vec<flume::Receiver<LinkMessage>>,
```

Comment:
Why is not using the `Input`?

Reply:
It's an oversight, I introduced the `InputRaw` and `OutputRaw` after making that change & forgot to port it